plenum/server/monitor.py

```diff
@@ -26,6 +26,51 @@
 logger = getlogger()
 
 
+class RequestMeasurement:
+    """
+    Measure request params (throughput, latency)
+    """
+
+    def __init__(self, inner_window=15, min_cnt=16):
+        self.reqs_in_window = 0
+        self.throughput = 0
+        self.inner_window = inner_window
+        self.min_cnt = min_cnt
+        self.first_ts = time.perf_counter()
+        self.window_start_ts = self.first_ts
+        self.alpha = 2 / (self.min_cnt + 1)
+
+    def add_request(self, ordered_ts):
+        self.moving_window(ordered_ts)
+        if ordered_ts >= self.window_start_ts and ordered_ts <= self.window_start_ts + self.inner_window:
+            self.reqs_in_window += 1
+
+    def moving_average(self, next_val):
```

Comment:
This is implementation details, probably better start it's name with `_` and move it down to improve readability